src/fontra/views/editor/scene-controller.js

```diff
@@ -641,6 +641,12 @@ export class SceneController {
 
   getContextMenuItems(event) {
     const contextMenuItems = [
+      {
+        title: translate("action.close-contour"),
+        enabled: () => this.doCloseContour(),
```

Comment:
The naming convention is (you can see that in all examples):
- enabled: canBlaBla
- callback: doBlaBla

Reply:
Btw. please look into `translatePlural`, as used with `action.decompose-component`. We need it here, too.

Reply:
Sorry about the first comment: it missed that that convention is only held in editor.js, not elsewhere, for better or for worse.